.github/workflows/ci.yaml

```diff
@@ -22,13 +21,11 @@ jobs:
       - run: rustup default nightly-2020-08-27
       - run: rustup component add rustfmt
       - run: rustup component add rust-src
+      - run: git clone --depth 1 --branch 0.1.36 https://github.com/rust-lang/compiler-builtins.git $(rustc --print sysroot)/lib/rustlib/src/compiler-builtins
```

Comment:
Will anyone using this need to do this themselves? this seems like it'll make it very inaccessible.

Reply:
Yeah, it is a requirement -- I noted it in the quick start. Why do you think it would be inaccessible? If you are referring to the `--depth`/`--branch`, that is something for the CI only, in the quick start I only mention cloning and making a symlink (same as you need to do for the `rust` repo if one doesn't use `rustup`).

Reply:
Improved the quick start guide even further, and also gave a one-shot command that can be copy-pasted (i.e. should simplify things further).

Reply:
Should we provide a script to do this, so the user does not have to open the quick start guide?

Reply:
Yeah, we can provide one to setup everything (skipping steps if something is found etc.). It may be worth it for the first year or so after landing, even if eventually (I imagine) distros will be providing the right environment out of the box in their kernel-dev packages.